.travis.yml

```diff
@@ -1,12 +1,19 @@
-# Use container-based infrastructure
 sudo: false
 
 language: scala
 
 scala:
-- 2.11.6
+  - 2.11.7
 
 jdk:
-- oraclejdk8
+  - oraclejdk8
 
-script: "sbt ++$TRAVIS_SCALA_VERSION clean test"
+env:
+  global:
+    - secure: "c3gMioUuhU2iiqmV0jx+mig20j0Fu/MLy/h9OCDQfsMi1PqH1woNospBPhvwjQgTm1Wk7ciC8vmYfc9oHVw5rejjKMAgPTgft3NsTXBXoJ4B1RAWFAm5Fg6ERCwxBC+F61HOroye5QbMI5mhdUNp4rrT4e17VqHcNzyuUrolqdw="
```

Comment:
Yup. I saw other projects do the same thing and thought it was a good best practice.